.dagger/src/index.ts

```
@@ -0,0 +1,1258 @@
+          result.returns.deployed.value
+        ) {
+          contractAddress = result.returns.deployed.value
+        }
+
+        // Extract constructor args from `.returns.constructorArgs.value`
+        if (
+          result.returns &&
+          result.returns.constructorArgs &&
+          result.returns.constructorArgs.value
+        ) {
+          constructorArgs = result.returns.constructorArgs.value
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)